src/libraries/Microsoft.Extensions.Primitives/src/StringTokenizer.cs

```diff
@@ -1,7 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
```

Comment:
Can you undo this unnecessary change? That way it doesn't pollute source control history.